src/hooks/use-input.ts

```diff
@@ -78,6 +79,16 @@ export interface Key {
 
 type Handler = (input: string, key: Key) => void;
 
+// Taken from https://nodejs.org/api/readline.html#readline_rl_write_data_key
+interface ReadlineKeyInfo {
```

Comment:
Can you not import it from the Node.js types?

Reply:
I assumed they'd exist aroudn the `keypress` event but they were in `readline` and I missed them, so will updated 👍